ui-admin/package.json

```diff
@@ -33,7 +35,8 @@
     "@types/marked": "^4.0.7",
     "@types/react": "^18.0.18",
     "@types/react-dom": "^18.0.6",
-    "@types/react-table": "^7.7.12"
+    "@types/react-table": "^7.7.12",
+    "canvas": "^2.11.2"
```

Comment:
I didn't see this used anywhere? Is it a peer dependency of plotly? If so, it would be preferable to put it in dependencies instead of dev dependencies.

Reply:
yeah, this is confusing -- and it's where I really wish you could add comments in package.json.  this is only needed for plotly jest tests.  Otherwise, the test console fills up with "...canvas" is not a function warnings.  https://github.com/hustcc/jest-canvas-mock used to let you get around this more cleanly, but there are compatibility issues between that and later versions of create-react-app